File: scraping-kotlin/src/main/kotlin/utils/FileUtil.kt
package utils

import java.nio.file.Files
import java.nio.file.Path
import java.nio.file.StandardOpenOption
import java.time.LocalDate
import java.time.LocalDateTime

class FileUtil {

    companion object {
        val FILE_PATH = "/Users/yangsiyeong/study/scraping-kotlin/src/main/resources/files"

        fun saveAs(content: String, fileName: String): String {
            val currentDate = LocalDate.now()
            val saveFileName = fileName + currentDate
            val path = Path.of("$FILE_PATH/$saveFileName.txt")
            val option = if(Files.exists(path)) StandardOpenOption.APPEND else StandardOpenOption.CREATE
            Files.write(path, content.encodeToByteArray(), option)
            return path.toString()
        }
    }
}

File: scraping-python/utils/FileUtil.py
from datetime import datetime


class FileUtil:

    def save_as(self, content, file_name):
        current_date = datetime.today().strftime("%Y%m%d")
        content = "\n\n" + content
        # file = open('/Users/yangsiyeong/study/scraping-python/files/' + file_name + current_date + ".txt", "a")
        file = open('/Users/yangsiyeong/study/scraping-assignment/scraping-python/files/' + file_name + current_date + ".txt", "a")
        file.write(content)
        file.flush()
        file.close()
